ci/run-docker.sh

```diff
@@ -64,6 +64,21 @@ case $TARGET in
     OPENSSL_CC=x86_64-unknown-netbsd-gcc
     OPENSSL_AR=x86_64-unknown-netbsd-ar
     ;;
+  powerpc-*-linux-*)
+    OPENSSL_OS=linux-ppc
+    OPENSSL_CC=powerpc-linux-gnu-gcc
+    OPENSSL_AR=powerpc-linux-gnu-ar
+    ;;
+  powerpc64-*-linux-*)
+    OPENSSL_OS=linux-ppc64
+    OPENSSL_CC=powerpc64-linux-gnu-gcc
```

Comment:
Oh the failure now is:

```
powerpc64-linux-gnu-gcc: Command not found
```

I think that powerpc64 works by using the `powerpc-linux-gnu` compiler and then passing `-m64` as a flag